urllib3/connection.py

```diff
@@ -62,6 +57,14 @@ class ConnectionError(Exception):
 RECENT_DATE = datetime.date(2014, 1, 1)
 
 
+class DummyConnection(object):
+    "Used to detect a failed ConnectionCls import."
+    pass
+
+
+HTTPSConnection = DummyConnection
```

Comment:
I was referring to the class `HTTPSConnection` defined on line ~168, which is the parent class of `VerifiedHTTPSConnection`.

on lines ~279 the name `HTTPSConnection` is assigned the value of `VerifiedHTTPSConnection` or `DummyConnection`, and therefore the class defined on ~168 is hidden except in the bases of `VerifiedHTTPSConnection`.  That is a little confusing.(and causes conversations like this one.)
only the value assigned on lines ~279 is part of the public API.